cdap-cli/src/main/java/co/cask/cdap/cli/util/ArgumentParser.java

```diff
@@ -47,7 +47,7 @@
     if (mapString == null || mapString.isEmpty()) {
       return ImmutableMap.of();
     }
-    return Splitter.on(" ").withKeyValueSeparator("=").split(mapString);
+    return Splitter.on(",").omitEmptyStrings().trimResults().withKeyValueSeparator("=").split(mapString);
```

Comment:
@nitinmotgi suggested that we need to allow spaces in values. For example with the splitting strategy used before, we cannot have "name=nitin motgi". Hence the reason to split by ",".